Charles/2025 swe hiring (#1104)
arXiv/arxiv-docs: source/hiring/index.md

 - improve the metadata we collect and make available for articles, adding fields that the research community has requested such as funder identification
 - deal with the problem of ambiguous author identities
 - improve accessibility to support users with impairments, particularly visual impairments
 - improve usability for the entire arXiv community

We have three SWE openings, one general and two specialized

###  Software Engineer

This person should be a software generalist, who can work in a variety of environments, programming languages, frontend and backend. This job is listed at three different levels. Candidates should examine each job description, in terms of education and experience required, to determine the appropriate level to apply at. Please avoid applying at more than one level. If you are otherwise qualified for the position, but applied at the wrong level, we'll still consider your application, but might suggest switching levels.

 - [Software Engineer III](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
 - [Software Engineer IV](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv) (Software Engineer)

>  - [Software Engineer V](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv) (Senior Software Engineer)

Reviewer: Only level IV are posted, will the III and V follow, or does this text need updating to IV-only?

# Careers at arXiv

 arXiv is part of Cornell Tech, the graduate campus and research center of Cornell University. All arXiv employees are Cornell University employees.

 - Positions at arXiv are [staff positions with benefits (endowed)](https://hr.cornell.edu/understand-your-benefits).

>  - Visa sponsorship is not available. You must already posses the legal right to work in the US.

Reviewer: ```suggestion
 - Visa sponsorship is not available. You must already possess the legal right to work in the US.
```

 - replace the portion of our backends still written in perl and PHP
 - re-architect our article processing to be fully asynchronous, and provide better insight into the processing workflows
 - containerize all, or nearly all arXiv services so we can deploy via Kubernetes or services like [Google Cloud Run](https://cloud.google.com/run/)
 - improve our monitoring and logging facilities so we can more quickly identify and manage production issues with arxiv.org
 - create a robust CI/CD pipeline to give us more confidence that changes we deploy will not cause services to regress

The cloud transition is a pre-requisite to modernizing arXiv as a service. The modernization will enable:
 - arXiv to expand the subject areas that we cover
 - improve the metadata we collect and make available for articles, adding fields that the research community has requested such as funder identification
 - deal with the problem of ambiguous author identities
 - improve accessibility to support users with impairments, particularly visual impairments
 - improve usability for the entire arXiv community

We have three SWE openings, one general and two specialized

###  Software Engineer

> This person should be a software generalist, who can work in a variety of environments, programming languages, first-end and backend. This job is listed at three different levels. Candidates should examine each job descriptions, in terms of education and experience required, to determine the appropriate level to apply at. Please avoid applying at more than one level. If you othewise qualfied for the position, but applied at the wrong level, we'll still consider your application, but might suggest switching levels.

Reviewer: ```suggestion
This person should be a software generalist, who can work in a variety of environments, programming languages, frontend and backend. This job is listed at three different levels. Candidates should examine each job description, in terms of education and experience required, to determine the appropriate level to apply at. Please avoid applying at more than one level. If you are otherwise qualified for the position, but applied at the wrong level, we'll still consider your application, but might suggest switching levels.
```

 - [Software Engineer III](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
 - [Software Engineer IV](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv) (Software Engineer)
 - [Software Engineer V](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv) (Senior Software Engineer)

> ### DevOps specialist

Reviewer: ```suggestion
### DevOps Specialist
```

 We are looking for a developer with Cloud DevOps experience to lead the effort of modernizing our devops system so we can efficiently, reliably, and securely deploy new code, using Infrastructure as code principles. This task will likely involve a fair amount of automation via [gitops](https://github.com/readme/featured/defining-gitops), as well as Google Cloud Platform tools. The devops lead will also put in place a robust CI/CD pipeline, and may also be asked to help fill in test cases.

>  itions.

Reviewer: ```suggestion

```

 - [DevOps Software Engineer](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)

### Scientist/Software Developer

> We're looking for someone with seroius software developmenet skills who has also used arXiv for scientific research, and preferably someone who has authored arXiv submissions. We expect this person to represent and be an advocate for the interests all scientists, within the arXiv software development team (no pressure!). We believe there are lots of you scientists-turned-software-developers out there. Please apply!

Reviewer: ```suggestion
We're looking for someone with serious software development skills who has also used arXiv for scientific research, and preferably someone who has authored arXiv submissions. We expect this person to represent and be an advocate for the interests of all scientists, within the arXiv software development team (no pressure!). We believe there are lots of you scientists-turned-software-developers out there. Please apply!
```

> - [Software Engineer Scientist](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)

Reviewer: ```suggestion
- [Scientist/Software Developer](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
```
